ctre/_impl/sim_ui.py

```diff
@@ -68,30 +68,32 @@ def _add_CAN(self, sim, canId, device):
 
         # TODO: this is not flexible
 
-        row = len(self.can) * 2
+        lcan = len(self.can)
+        row = (lcan % 6) * 2
+        col = (lcan // 6) * 5
```

Comment:
Wonder whether using `divmod` here would be better. I guess it'd end up being longer though.